build/common/config/.golangci.yml

```diff
@@ -80,7 +80,7 @@ linters-settings:
     check-shadowing: false
   gci:
     sections:
-      - prefix(open-cluster-management.io/config-policy-controller)
+      - prefix(open-cluster-management.io/governance-policy-propagator)
```

Comment:
😹

Reply:
Yeah--makes me wonder whether this happens elsewhere! 😢

Reply:
This is probably the config file I was thinking of... I need more coffee 😂